florist/tests/integration/api/utils.py

```diff
@@ -0,0 +1,21 @@
+import contextlib
+import time
+import threading
+import uvicorn
+
+
+class Server(uvicorn.Server):
```

Comment:
Should we add a more descriptive name maybe (UvicornServer) to avoid confusing with server from FL terminology